### ISSUE TITLE
Enhancement: Require Base Styles of Certain Categories to be Selected via Drop Down List

### ISSUE THREAD

**Issue by 003401E8** [geoffhumphrey/brewcompetitiononlineentry#1515]
**Reviewed the Release Notes?**
Yes

**Is your enhancement or feature request related to a problem? Please describe.**
Yes. The problem with Category 30 (and others) entries is the loophole that is created based on the paragraphs shown. 

**Describe the solution you'd like:**
I'd like to see this reworked to have a dropdown that is required to have a base style selected for any category that requires this for judging. As is, the 30A category paragraph in BCOE&M is:
"The entrant MUST specify the type of spices, herbs, or vegetables used, but individual ingredients do not need to be specified if a well-known spice blend is used (e.g., apple pie spice, curry powder, chili powder). Entrant MUST specify a description of the beer, identifying either a Base Style or the ingredients, specs, or target character of the beer. A general description of the special nature of the beer can cover all the required items."

If I state "Pumpkin beer with pumpkin pie spices" in the text field, the loophole created is that it specifies the spices and a description of the beer, but does not give any clue to the judges on which base style it is, but you cannot penalize them because of the verbiage. A base style should HAVE to be selected via dropdown like when you select the category your entry will be in, with the text box being the space where they list the fruit, SHV, etc. 

![image](https://github.com/geoffhumphrey/brewcompetitiononlineentry/assets/71900912/de6eafef-a258-4917-a4ec-117b0a05fddb)
![image](https://github.com/geoffhumphrey/brewcompetitiononlineentry/assets/71900912/c51e417b-9c89-48ed-8e9d-e4319214041a)

**Describe alternatives you've considered:**
Writing very specific rules to try and avoid loopholes

**Additional context:**
N/A

**Comment by geoffhumphrey** [comment]
Hi @003401E8 and thanks for the suggestion. While I understand the reasons for your suggestion, I'm reticent in force users to choose a base style due to the second part of the clause that is directly derived from the BJCP styles: "_either_ a Base Style or the ingredients, specs, or target character of the beer" (emphasis is mine).

I see the point of trying to get the most accurate information possible, but since the BJCP specifies two ways to get at the target characteristics of the entry, the inclusion of a drop-down could serve to pigeon-hole entrants into declaring a style when they are not required to do so. 

Boils down to garbage in = garbage out. If an entrant does not supply the required information, their entry should be judged accordingly. Specifically, in your example, Pumpkin Beer is not a sufficient enough base style description and judges who evaluate that entry should not only note the lack of sufficient information, but also should judge the beer according to what they're given without speculating on the entrants intended base style.

Of course, this is my opinion. If there's sufficient interest from the community, we can definitely explore options for a future release.

**Comment by jimmyorkin** [comment]
I agree with Geoff. It is up the the entrant to provide the correct information, not the organizers.

**Comment by 003401E8** [comment]
How about adding 'N/A' or 'Other' to the top of the drop down list so it puts the onus on the entrant to specify and would still require a description in the field?
